node-tests/fixtures/expected-manifests.json

```diff
@@ -11,10 +11,6 @@
     "bundles": {
       "lazy": {
         "assets": [
-          {
```

Comment:
@stefanpenner this lines removed no longer exist on ember engines build?

Reply:
this was just local testing that I committed. We are good